src/Store/Reducers/animationReducer.js

```diff
@@ -0,0 +1,34 @@
+const initialState = [];
+
+function animationReducer(state = initialState, action) {
+  let nextState;
+  switch (action.type) {
+    case 'SAVE_ANIMATION':
+      // If the saved animation previously had another title, remove the animation with this old title
+      let oldAnimationIndex = state.findIndex(item => item.title === action.value.oldTitle);
```

Comment:
It smells like this should be a "renameDrill" action since we do 2 different things here ?